src/components/sound.js

```diff
@@ -70,7 +70,7 @@ module.exports.Component = registerComponent('sound', {
   },
 
   remove: function () {
-    this.el.object3D.remove(this.sound);
+    this.el.setObject3D('sound', null);
```

Comment:
kinda feel like a `removeObject3D` would be more explicit and clear in the long run